README.md

```diff
@@ -37,21 +37,18 @@ Now, you can use that URL when the script asks you to input, which will be used
 You will need a node.js and yarn installed in your system.
 
 You need a few tools to be available in your terminal:
- - `cardani-cli`
- - `cardano-address`
- - `bech32`
+ - `cardani-cli` (you can get it from Daedalus.)
```

Comment:
```suggestion
 - `cardano-cli` (you can get it from Daedalus.)
```

Reply:
thank you :) fixed here [6f8e7a1d8b6972922a7143f6a860a4f126218113](https://github.com/input-output-hk/dapp-store-cip72/pull/12/commits/6f8e7a1d8b6972922a7143f6a860a4f126218113)